From: Alexander Duyck <alexander.h.duyck@redhat.com>
Date: Thu, 9 Apr 2015 22:03:24 -0700
Subject: [PATCH 154/177] ixgbe: Drop unnecessary call to rcu_barrier
Patch-mainline: v4.1-rc1
Git-commit: 020f1d9d9ae3f476d887f2b043223d8a89652174
References: bsc#922833 FATE#318521

The ixgbe driver hasn't used call_rcu to free the rings for some time now.
Since that is the case the call to rcu_barrier can be dropped since calls
to kfree_rcu don't require it.

Signed-off-by: Alexander Duyck <alexander.h.duyck@redhat.com>
Tested-by: Phil Schmitt <phillip.j.schmitt@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Acked-by: Cho, Yu-Chen <acho@suse.com>
---
 drivers/net/ethernet/intel/ixgbe/ixgbe_main.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/net/ethernet/intel/ixgbe/ixgbe_main.c
+++ b/drivers/net/ethernet/intel/ixgbe/ixgbe_main.c
@@ -8944,8 +8944,6 @@ static void __exit ixgbe_exit_module(voi
 	pci_unregister_driver(&ixgbe_driver);
 
 	ixgbe_dbg_exit();
-
-	rcu_barrier(); /* Wait for completion of call_rcu()'s */
 }
 
 #ifdef CONFIG_IXGBE_DCA
